### DISCUSSION TITLE
b站首页个性化推荐很恶心，希望回到原版首页视频推荐

### DISCUSSION THREAD

**Discussion by 2twothree** [the1812/Bilibili-Evolved#1885] [功能建议 / Ideas]
不知道以后能不能添加一个还原原版首页的功能，在网页端登录之后显示的视频推荐是个性化的，很恶心，不登录之后就显示出以前的每个用户都一样的视频推荐，希望能够支持，谢谢

**Comment by 2twothree** [comment]
https://greasyfork.org/zh-CN/scripts/425943-%E7%A6%81%E7%94%A8b%E7%AB%99%E9%A6%96%E9%A1%B5%E9%A6%96%E6%A0%8F%E4%B8%AA%E6%80%A7%E6%8E%A8%E8%8D%90

这个脚本能用，但是不能翻页，不够完善

**Comment by Geeyun-JY3** [comment]
请不要开重复的主题：#1842
